Extensions/read_more_now.js

```diff
@@ -1,211 +1,104 @@
 //* TITLE Read More Now **//
-//* VERSION 1.4.7 **//
+//* VERSION 2.0.0 **//
 //* DESCRIPTION Read Mores in your dash **//
-//* DETAILS This extension allows you to read 'Read More' posts without leaving your dash. Just click on the 'Read More Now!' button on posts and XKit will automatically load and display the post on your dashboard. **//
-//* DEVELOPER STUDIOXENIX **//
+//* DETAILS This extension changes &quot;Keep Reading&quot; links into collapsible elements for easy viewing. **//
```

Comment:
I'm not 100% confident on the details element change as a UI change—I don't want to break people's assumptions about how XKit's interface works or make them think Read More Now has stopped working. Also, under the old design you could choose whether to click the keep reading link or the read more now button, which you can't with the new extension.

Reply:
i have to admit i have a personal bias towards integrating XKit with Tumblr's interface more closely but you're completely right. I'll change the UI and fetch time back to how it used to work